2020/13xxx/CVE-2020-13670.json

```diff
@@ -1,18 +1,74 @@
 {
```

Comment:
{} is not allowed for {'problemtype_data': [{'description': [{'lang': 'eng', 'value': 'Information Disclosure'}]}]}